### DISCUSSION TITLE
Is there a list of announcements & details ?

### DISCUSSION THREAD

**Discussion by Josverl** [community/events#29] [Keynotes]
in general,
and specifically I would like to understand what options there are for OSS repo maintainers to use codespaces ?

**Comment by ossanna16** [comment]
Hi there @Josverl! Can you elaborate a bit further on your question? What's the specific use case you have in mind?

**Comment by Josverl** [comment]
for the general case I was hoping that there would be a overview of the main items just announced in the keynote.

For Codespaces, if I understood correctly then all developers can use Codespace for up to 60 hrs/month

but I also heard that there we extra allowances for maintainers of public repos, and I was wondering if that may mean that some of my favorite maintainers would be allowed more.

**Reply by ossanna16** [reply]
You can find a summary of the keynote announcements in this [blog post](https://github.blog/2022-11-09-everything-new-from-github-universe-2022/).

**Reply by otter-computer** [reply]
Specifically for your ask on Codespaces, the Codespaces team are currently working on providing storage for [prebuilds](https://docs.github.com/en/codespaces/prebuilding-your-codespaces/about-github-codespaces-prebuilds) for OSS projects allowing users to spin up and work on these projects in a codespace quickly, but right now there isn't anything further to announce on this. Keep an eye on the GitHub blog for news on Codespaces for OSS projects.

**Comment by jricciardi** [comment]
For the sake of a clean answer to mark: https://github.blog/2022-11-09-everything-new-from-github-universe-2022/ :)
